source/clear-linux/guides/network/network-bonding.rst

```diff
@@ -86,25 +91,25 @@ __ https://www.freedesktop.org/software/systemd/man/systemd.netdev.html#id-1.20.
       [Link]
       MTUBytes=9000
 
-   Since ``bond1`` is a virtual interface, it has no concept of physical link
-   status. The ``BindCarrier`` directive indicates that the link status of this
-   interface is determined by the status of the listed slave devices.
+   `bond1` is a virtual interface with no physical link status.
 
-   This is the logical interface, so assign it an IP address. DHCP is more
-   complicated with bonded interfaces, and is not covered in this example.
+   `BindCarrier` indicates that the `bond1` link status is determined by the
+   status of the listed slave devices.
 
-   This file also enables jumbo frames of up to 9000 bytes. This option must be
-   enabled for all slave interfaces *and* the bonded interface, in order to take
-   effect.
+   `Address` contains an IP address that you assign to the logical interface.
+   DHCP bonded interfaces are complex and outside the scope of this example.
 
-5. Apply the new network configuration:
+   `MTUBytes` must be the same on all slave interfaces and on the bonded
```

Comment:
I think this needs to be expanded a little bit. Technically, the lowest value configured will be used across all interfaces; they don't have to be the same. But the default value is 1500, so if you want 9000 (jumbo), it must be set for all slaves plus the bonded interface.

Reply:
Thanks for the feedback. Please let me know if the updated text is OK.